go/tendermint/tendermint.go

```diff
@@ -515,15 +519,28 @@ func (t *tendermintService) lazyInit() error {
 	tenderConfig.P2P.ListenAddress = viper.GetString(CfgCoreListenAddress)
 	tenderConfig.P2P.ExternalAddress = viper.GetString(cfgCoreExternalAddress)
 	tenderConfig.P2P.AllowDuplicateIP = true // HACK: e2e tests need this.
+	tenderConfig.P2P.PrivatePeerIDs = viper.GetString(CfgP2PPrivatePeers)
+	// Persistent peers need to be lowecase as p2p/transport.go:MultiplexTransport.upgrade()
+	// uses a case sensitive string comparision to validate public keys.
+	// Since persistent peers is expected to be in comma-delimited id@host:port format,
```

Comment:
similarly as with `CfgP2PSeeds` we maybe want to define `CfgP2PPrivatePeers` as a s string slice argument

Reply:
Done.